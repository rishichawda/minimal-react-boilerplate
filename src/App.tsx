import React from 'react';

import './App.scss';

const App = () => (
  <div className="app">
    <h2>Welcome to React!</h2>
  </div>
)

export default App;
