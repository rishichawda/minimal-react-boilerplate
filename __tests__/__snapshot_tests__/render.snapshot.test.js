/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'
import App from 'App'

test('should render App', () => {
  const app = renderer.create(<App />)
  const domTree = app.toJSON()
  expect(domTree).toMatchSnapshot()
})
