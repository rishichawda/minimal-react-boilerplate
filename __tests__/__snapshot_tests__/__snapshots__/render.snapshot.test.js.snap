// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App 1`] = `
<div
  className="app"
>
  <h2>
    Welcome to React!
  </h2>
</div>
`;
